tommath.h

```diff
@@ -729,10 +729,10 @@ mp_err mp_fwrite(const mp_int *a, int radix, FILE *stream) MP_WUR;
 #define mp_todecimal(M, S) (MP_DEPRECATED_PRAGMA("replaced by mp_to_decimal") mp_toradix((M), (S), 10))
 #define mp_tohex(M, S)     (MP_DEPRECATED_PRAGMA("replaced by mp_to_hex")     mp_toradix((M), (S), 16))
 
-#define mp_to_binary(M, S, N)  mp_to_radix((M), (S), (N), 2)
-#define mp_to_octal(M, S, N)   mp_to_radix((M), (S), (N), 8)
-#define mp_to_decimal(M, S, N) mp_to_radix((M), (S), (N), 10)
-#define mp_to_hex(M, S, N)     mp_to_radix((M), (S), (N), 16)
+#define mp_to_binary(M, S, N)  mp_to_radix((M), (S), (N), NULL, 2)
+#define mp_to_octal(M, S, N)   mp_to_radix((M), (S), (N), NULL, 8)
+#define mp_to_decimal(M, S, N) mp_to_radix((M), (S), (N), NULL, 10)
+#define mp_to_hex(M, S, N)     mp_to_radix((M), (S), (N), NULL, 16)
```

Comment:
M, S, N, W?

Reply:
(The squash I just did is just a clean-up, not an end-it-all!)

> M, S, N, W?
Yeah, that's a good question.
I didn't add it at the end because it is a macro that is meant, in my opinion, to be a *simple* shortcut with intentionally reduced functionality. But if you want it: no problem.

Reply:
I already thought about overloading macros with different number of arguments, but unfortunately that's only doable for gcc etc. ... I could live with both MSN or MSNW

Reply:
Ok, @czurnieden is right. The written argument can be omitted here for the simplified/specialised macros.